.github/ISSUE_TEMPLATE/request_for_ci.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Request for including into CI
+description: Use this template to create a request for including a software tool into CI pipelines per facility.
+title: "[CI Request]: [SOFTWARE TOOL NAME]"
+labels: "Continuous Integration"
+assignees: "mtitov, danlaney"
+---
+
+Please fullfil the initial requirements from the following document: 
+[Contributing to SDK](https://exaworkssdk.readthedocs.io/en/latest/contribute.html).
+
+1. **CI Request** includes the following information about the software tool:
+
+- [ ] Description of the tool
+- [ ] GitHub repository (URL)
+- [ ] Supported package managers: **pip**, **conda**, **spack**
+- [ ] Version to be used: **specific** or **the latest**
```

Comment:
We can allow for the latest but need at least one concrete version number?

Reply:
updated - "latest" will be optional, but providing a specific version is required